packages/go_router/README.md

```diff
@@ -36,7 +36,8 @@ See the API documentation for details on the following topics:
 - [Named routes](https://pub.dev/documentation/go_router/latest/topics/Named%20routes-topic.html)
 - [Error handling](https://pub.dev/documentation/go_router/latest/topics/Error%20handling-topic.html)
 
-## Migration guides
+## Migration Guides
```

Comment:
link  404 not found

Reply:
Need to wait for this pr https://github.com/flutter/website/pull/9092

Reply:
pr merged